### ISSUE TITLE
Add F2 keyboard shortcut to edit text boxes

### ISSUE THREAD

**Issue by zapatacarlos** [excalidraw/excalidraw#3707]
Hi! It would be really practical to implement the F2 keyboard shortcut for when one grabs a text box and wants to edit it. It is a very common shortcut and avoids changing to the mouse for the double click already available. Very happy with Excalidraw, great tool!

**Comment by dwelle** [comment]
In the meantime, you can also use `enter` to the edit selected text element.

**Comment by drm** [comment]
Yes! F2 is the "de-facto" shortcut for this, especially when you're used to working with data (excel and the likes).

**Comment by dwelle** [comment]
We could do it, but since text editing is as simple as using `enter`, we thought that `f2` could be used in Excalidraw+ to edit scene title instead (or scene title in excalidraw.com as well, when later saving to a file).

**Comment by limex** [comment]
F2 would cause conflict in Obsidian, I guess.

**Comment by nlwillia** [comment]
> We could do it, but since text editing is as simple as using `enter`, we thought that `f2` could be used in Excalidraw+ to edit scene title instead (or scene title in excalidraw.com as well, when later saving to a file).

F2 is ubiquitous for edit/rename of the local element.  I was surprised that it didn't "just work" and came looking for this suggestion, which is friction I hope your designers will consider.
